python-example-noncontainer-artifacts/cloudbuild.yaml

```diff
@@ -12,24 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # [START cloudbuild_python_yaml]
+
+# [START cloudbuild_python_dependencies_yaml]
+# Install dependencies
```

Comment:
We generally prefer not to have language-specific region tags, because in most cases we try to have the use cases/user journeys/concepts explicitly covered in multiple languages and our tooling does various kinds of grouping by region tags to show gaps, metrics, and so on.

Could we make this `cloudbuild_dependencies_yaml` instead, and extend that concept to the other region tags?